### ISSUE TITLE
1Remote is started as admin whenever automatically run at OS startup

### ISSUE THREAD

**Issue by b-a0** [1Remote/1Remote#811]
**Describe the bug**
When the checkbox "Startup: Run automatically at OS startup(minimized)" is checked 1Remote starts, but has elevated permissions. Other applications started by 1Remote (e.g. Windows Terminal with an SSH session) are also run as admin.

**To Reproduce**
1. Go to 'Options'
2. Check 'Run automatically at OS startup(minimized)'
3. Logout of Windows and log back in
4. Wait until 1Remote start
5. Observe in Task Manager that the 1Remote.exe process has "Yes" in the "Elevated" column.

**Expected behavior**
I would expect that 1Remote is run under the current user as a regular, non-elevated process, similar to what happens when you simply launch 1Remote yourself.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11 Version 23H2
 - 1RemoteM version: 1.0.1-alpha build at 2024-12-07T01:14:50.657 (installed from [chocolatey](https://community.chocolatey.org/packages/1remote))

**Comment by b-a0** [comment]
Related to this I have a question for a workaround:

I can disable the checkbox and use Task Scheduler to run 1Remote on login. But I cannot find a command line argument that runs 1Remote.exe minimized in the tray. Is there such an option?

**Comment by majkinetor** [comment]
https://1remote.org/usage/misc/cli/#startup-entry-management

**Comment by VShawn** [comment]
Did the Chocolatey package cause the privilege escalation operation? I never requested admin privileges for 1Remote.

**Comment by majkinetor** [comment]
It seems so, people usually run it as admins.

**Comment by majkinetor** [comment]
So, workaround would be to call this manually: `1remote.exe --install-startup`

**Comment by b-a0** [comment]
> Did the Chocolatey package cause the privilege escalation operation? I never requested admin privileges for 1Remote.

Yes, Chocolatey strongly advises that [it can install packages as admin](https://chocolatey.org/install#install-step2). This seems to be an issue with the chocolatey package then, not with 1Remote. I will create an issue with the package maintainer.

**Comment by majkinetor** [comment]
This is not about the package. It only unpacks the files and creates desktop icon, restarting the app if its running already.
